cmd/exec/operations.go

```diff
@@ -446,7 +446,7 @@ func executeEvent(url string) (string, error) {
 	inputDataOsFile, err := os.Open(localAbsPath)
 
 	if err != nil {
-		return "", err
+		log.Fatalf("Failed to load input file: %v", err)
```

Comment:
The function should return the error and escalate it to the caller. It should be the caller's decision whether to fail or try something else.